de.dlr.sc.virsat.model.extension.statemachines/src/de/dlr/sc/virsat/model/extension/statemachines/excel/importer/SMImporter.java

```diff
@@ -28,59 +28,51 @@
 import de.dlr.sc.virsat.model.extension.statemachines.model.StateMachine;
 import de.dlr.sc.virsat.model.extension.statemachines.model.Transition;
 
-
-
-
 /**
  * Class for Importing Excel files.
- * 
- * @author bell_er
- *
  */
 public class SMImporter implements IImport {
 	private XSSFWorkbook wb;
-	private StateMachine sm;
+	private StateMachine stateMaschine;
 	private Concept concept;
-	
+
 	@Override
 	public void importExcel(EObject eObject, Repository repository, XSSFWorkbook wb) {
 		init(eObject, repository, wb);
 		importStates();
 		importTransitions();
 	}
-	
+
 	/**
-	 * imports the transitions 
-	 * 
-	 * @author  Bell_er
+	 * imports the transitions
 	 */
	private void importTransitions() {

```

Comment:
Looks like there is a wrong line ending here, let's fix it

Reply:
hopefully done